Alignment/OfflineValidation/plugins/PrimaryVertexValidation.cc

```diff
@@ -283,10 +283,10 @@ void PrimaryVertexValidation::analyze(const edm::Event& iEvent, const edm::Event
   //edm::Handle<VertexCollection> vertices;
   edm::Handle<std::vector<Vertex>> vertices;
 
-  try {
-    vertices = iEvent.getHandle(theVertexCollectionToken_);
-  } catch (cms::Exception& er) {
-    LogTrace("PrimaryVertexValidation") << "caught std::exception " << er.what() << std::endl;
+  vertices = iEvent.getHandle(theVertexCollectionToken_);
+  if (!vertices.isValid()) {
+    LogTrace("PrimaryVertexValidation") << "Handle missing product" << std::endl;
```

Comment:
it would be nice to state what product is missing.

Reply:
In this case the earlier try-catch was effectively a no-op, because the `Event::getHandle()` does not throw an exception if the product is missing.

Reply:
> does not throw an exception if the product is missing.

This code is there since before the gitHub migration.

Reply:
@yuanchao as a result of the discussion, once the name of the missing product is added here the update can be merged. For example:
```suggestion
    LogTrace("PrimaryVertexValidation") << "Vertex collection Handle is missing product" << std::endl;
```
(probably a LogError would be more indicated here, but let the destination as originally intended to avoid more relevant changes to a code that was run apparently without problems till now)

Reply:
> can be merged 

There is still no master PR

Reply:
> > can be merged
> 
> There is still no master PR

Correct, thank you @mmusich !
Please @yuanchao also rebase and move it to master; or alternatively close this PR and open a new one in the master. There will be no need to backport, then